install.md

```diff
@@ -8,32 +8,52 @@ and verified the cluster, accepted the EULA, and installed the Tanzu CLI with an
 See [Installing the Tanzu CLI](install-tanzu-cli.md).
 
 
-## <a id='add-tap-package-repo'></a> Add the Tanzu Application Platform package repository
+## <a id='add-tap-package-repo'></a> Relocate Images to a Registry
 
-To add the Tanzu Application Platform package repository:
+Its recommended to relocate the Images to your registry from Tanzu Network registry before attempting installation.
+The supported conatiner registries are Harbor, Azure Container Registry, Google Container Registry, Qual.io. 
+Please refer to corresponding documentation on how to setup above listed registries.
 
-1. If you haven’t completed the
-[Install Cluster Essentials for VMware Tanzu for non-TKG clusters](install-tanzu-cli.md#tanzu-cluster-essentials)
-procedure, set up environment variables for use during the installation by running:
+This procedure relocates images from the Tanzu Network registry to your registry:
+
+1. Log in to your image registry:
 
     ```
-    export INSTALL_REGISTRY_USERNAME=TANZU-NET-USER
-    export INSTALL_REGISTRY_PASSWORD=TANZU-NET-PASSWORD
-    export INSTALL_REGISTRY_HOSTNAME=registry.tanzu.vmware.com
-    export TAP_VERSION=VERSION-NUMBER
+    docker login my.registry.io
+    ```
+    
+2. Log in to the Tanzu Network registry with your Tanzu Network credentials:
+
+    ```
+    docker login registry.tanzu.vmware.com
     ```
+    
+3. Set up environment variables for use during the installation by running:
 
+    ```
+    export INSTALL_REGISTRY_USERNAME=MY-REGISTRY-USER
+    export INSTALL_REGISTRY_PASSWORD=MY-REGISTRY-PASSWORD
+    export INSTALL_REGISTRY_HOSTNAME=my.registry.io
+    export TAP_VERSION=VERSION-NUMBER
+    ```
+    
     Where `VERSION-NUMBER` is your Tanzu Application Platform version. For example, `1.0.2`.
```

Comment:
Please provide information on what is MY-REGISTRY-USER, MY-REGISTRY-USER.
Chnage  to export INSTALL_REGISTRY_HOSTNAME= MY-REGISTRY so that user does not set his registry and not my.registry.io

Reply:
updated.